d2common/d2loader/filesystem/source.go

```diff
@@ -43,7 +43,7 @@ func (s *Source) fullPath(subPath string) string {
 	return filepath.Clean(filepath.Join(s.Root, subPath))
 }
 
-// Path returns the Root dir of this file system source
+// String returns the Root dir of this file system source
```

Comment:
Can you comment why this was changed? I suppose you replaced this by mistake?

Reply:
the change to the comment is a mistake

Reply:
I should note that I chose to have separate `String` and `Path` methods because I wanted it to implement `fmt.Stringer`, so if we use this object in something like `fmt.Sprintf` it will print the path of the source file.